### DISCUSSION TITLE
Argonone Gehäuse für RPi: Lüftersteuerung anpassen

### DISCUSSION THREAD

**Discussion by friedpa** [OpenCCU/OpenCCU#2835] [🇩🇪 Fragen&Antworten]
Wie kann man die Lüftersteuerung (die ja fix eingestellt wurde) ändern?
Das argononed.sh file ist (leider) Read-Only.
Hintergrund: das Gehäuse wird mir zu heiß....

**Comment by jens-maus** [comment]
Gar nicht aktuell. Kannst ja gerne nen PR schicken mit Anpassungen wo er die fan settings aus einer user datei von z.b. `/usr/local/etc/` einliest...

**Comment by friedpa** [comment]
Bin kein Developer, sorry....

**Comment by Baxxy13** [comment]
Ich hatte über 2 Jahre nen Pi4B im Argon-Gehäuse aktiv im Einsatz. Selbst bei max. 28°C Umgebungstemperatur wurde das Setup nie so warm das der Lüfter überhaupt angesprungen wäre.

Aber gut, es könnte ja auch ein hitzköpfiger Pi5B im Argon-Gehäuse sein... wer weiß das schon. 😉
